src/cases/components/role-exclusions/role-exclusions.component.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { Component, Input, ViewChild } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { RoleExclusion } from 'src/cases/models/role-exclusions/role-exclusion.model';
+import { RoleExclusionsComponent } from './role-exclusions.component';
+
+@Component({ template: `<exui-role-exclusions [allowDelete]="allowDelete" [exclusions]="exclusions"></exui-role-exclusions>` })
+
+class WrapperComponent {
+    @ViewChild(RoleExclusionsComponent) public appComponentRef: RoleExclusionsComponent;
+    @Input() public allowDelete: boolean;
+    @Input() public exclusions: RoleExclusion[];
+}
+
+describe('RoleExclusionsComponent', () => {
+    let component: RoleExclusionsComponent;
+    let wrapper: WrapperComponent;
+    let fixture: ComponentFixture<WrapperComponent>;
+
+    beforeEach((() => {
+        TestBed.configureTestingModule({
+            imports: [RouterTestingModule.withRoutes([]), HttpClientTestingModule],
+            declarations: [WrapperComponent, RoleExclusionsComponent],
+        }).compileComponents();
+        fixture = TestBed.createComponent(WrapperComponent);
+        wrapper = fixture.componentInstance;
+        component = wrapper.appComponentRef;
+        fixture.detectChanges();
+    }));
+
+    it('Is Truthy', () => {
+        expect(component).toBeTruthy();
+    });
+
+    it('Add link display', () => {
+        const element = fixture.debugElement.nativeElement;
+        const rowElement = element.querySelector('.govuk-link');
+        expect(rowElement.getAttribute('routerlink')).toEqual('/cases/addExclusion');
+    });
+
+    it('no exclusions message', () => {
```

Comment:
Messages should be more descriptive, e.g. it('should show the no exclusions message')

Reply:
Agree, changed

Reply:
Can you make the 'Add link display' more descriptive as well please? (currently I would read this as "It add link display" and that makes me think too much!)

Reply:
@paulhowes-HMCTS - renamed it to _Should always show the Add link display_

Reply:
Great.